src/calendar/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface CalendarProps {
+    disabledDates?: string[] | undefined;
```

Comment:
add jsdoc comment to show an example format

Reply:
the comment should be directly above the prop it is describing. you can reference `src/error-display/types.ts`